pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -394,6 +394,20 @@ else let
 
             [binaries]
             llvm-config = 'llvm-config-native'
+
+            ${lib.optionalString (stdenv.cc ? bintools && stdenv.cc.bintools.targetPrefix != "") ''
```

Comment:
Do we need to check `stdenv.cc.bintools.targetPrefix != ""`?

Reply:
not really, i just wanted to keep the meson file small but on second thought its better to keep the behaviour the same.